contrib/runners/python_runner/tests/unit/test_pythonrunner.py

```diff
@@ -667,6 +667,23 @@ def test_simple_action_log_messages_and_log_level_runner_param(self):
         self.assertFalse(expected_msg_4 in output['stderr'])
         self.assertTrue(expected_msg_5 in output['stderr'])
 
+        # Default log level is changed in st2.config
+        cfg.CONF.set_override(name='python_runner_log_level', override='INFO',
+                              group='actionrunner')
+
+        runner = self._get_mock_runner_obj()
+        runner.entry_point = PASCAL_ROW_ACTION_PATH
+        runner.runner_parameters = {}
+        runner.pre_run()
+        (status, output, _) = runner.run({'row_index': 'e'})
+        self.assertEqual(status, LIVEACTION_STATUS_SUCCEEDED)
+        self.assertTrue(output is not None)
+        self.assertEqual(output['result'], [1, 2])
+
+        self.assertTrue(expected_msg_3 in output['stderr'])
+        self.assertFalse(expected_msg_4 in output['stderr'])
```

Comment:
Not really important. But I would probably do `self.assertTrue(expected_msg_4 not in output['stderr'])` so to be consistent and show the message not in the stderr output.  This caught me off guard on first pass in the review.

Reply:
Yeah, this might be a bit of a personal preference, but I agree with you.